recipe/meta.yaml

```
@@ -45,7 +45,7 @@ requirements:
     - libitk
     - nlohmann_json
     - libparallelproj
-    - root_base  # [not win and not py==312]
+    - root_base  # [not win and not py==312 and not osx-arm64]
```

@h-vetinari I'm trying to exclude the problematic build, but failing. sorry. Maybe you can help?

---

You could look something like this
```suggestion
    # osx-arm skipped due to https://github.com/conda-forge/root-feedstock/issues/260
    - root_base  # [not (py==312 or win or (osx and arm64))]
```
This has more brackets but is IMO easier to understand, e.g. how it's naturally "skip if (X of Y of Z)", and also w.r.t. operator precedence. The issue is that the available selectors do not match the available target names, partly also because things get interpreted like python, so `osx-arm64` is a substraction.